content/post/80-js-ipfs-0.35.md

```diff
@@ -0,0 +1,240 @@
+* [Perry Kundert](https://github.com/pjkundert) (2 PRs)
+* [Pius Nyakoojo](https://github.com/PiusNyakoojo) (1 PR)
+* [Portia Burton](https://github.com/pkafei) (1 PR, 3 comments)
+* [postables](https://github.com/postables) (1 issue)
+* [Prabhat](https://github.com/jollysean) (3 comments)
+* [pranav maheshwari](https://github.com/pranavdaa) (1 issue, 2 comments)
+* [projectoblio](https://github.com/projectoblio) (1 issue, 2 comments)
+* [Rafael Matias](https://github.com/skylenet) (1 PR)
+* [Raúl Kripalani](https://github.com/raulk) (2 issues, 9 comments)
+* [Richard Schneider](https://github.com/richardschneider) (1 comment)
+* [Robert Kiel](https://github.com/robertkiel) (1 PR, 1 comment)
+* [Robert Misiorowski](https://github.com/rmisio) (4 issues, 7 comments)
+* [Rod Vagg](https://github.com/rvagg) (1 PR, 9 reviews, 6 comments)
+* [Rômulo Alves](https://github.com/romuloalves) (2 comments)
+* [rori4](https://github.com/rori4) (1 issue, 2 comments)
+* [Sam Strauch](https://github.com/SamTS) (1 issue)
+* [Sameer Puri](https://github.com/sameer) (2 PRs, 2 reviews, 4 comments)
+* [Steven Allen](https://github.com/Stebalien) (2 PRs, 2 comments)
+* [Taaliman](https://github.com/taaliman) (1 issue, 1 comment)
+* [Teri Chadbourne](https://github.com/terichadbourne) (1 issue, 4 comments)
+* [Theo Gravity](https://github.com/theogravity) (1 comment)
+* [Thomas Eizinger](https://github.com/thomaseizinger) (3 PRs, 1 issue, 8 comments)
+* [Tim Schuppener](https://github.com/ultraschuppi) (1 issue, 2 comments)
+* [Tyler van der Hoeven](https://github.com/tyvdh) (1 comment)
+* [ukrocks007](https://github.com/ukrocks007) (1 issue)
+* [useaquestion](https://github.com/useaquestion) (1 issue, 2 comments)
+* [UtopiaCreatorX](https://github.com/UtopiaCreatorX) (1 issue, 2 comments)
+* [vasa](https://github.com/vasa-develop) (1 issue, 3 comments)
+* [Vasco Santos](https://github.com/vasco-santos) (26 PRs, 7 issues, 83 reviews, 43 comments)
+* [Victor Bjelkholm](https://github.com/victorb) (1 review, 1 comment)
+* [Vincent Martin](https://github.com/vincepmartin) (2 comments)
+* [Vishal1010101](https://github.com/Vishal1010101) (1 comment)
+* [Volker Mische](https://github.com/vmx) (30 PRs, 3 issues, 65 reviews, 76 comments)
+* [w3cshare](https://github.com/w3cshare) (1 comment)
+* [Whyrusleeping](https://github.com/whyrusleeping) (1 issue)
+* [X5 Engine](https://github.com/x5engine) (1 issue, 4 comments)
+* [xialvjun](https://github.com/xialvjun) (1 issue)
+* [Yusef Napora](https://github.com/yusefnapora) (2 PRs, 1 comment)
+* [yuwiggin](https://github.com/yuwiggin) (1 issue, 1 comment)
+* [Zane Starr](https://github.com/zcstarr) (2 PRs, 2 reviews)
+
+# 🙌🏽 Want to contribute?
+
+Would you like to contribute to the IPFS project and don't know how? Well, there are a few places you can get started:
+
+- Check the issues with the `help wanted` label at the Ready column in our waffle board - https://waffle.io/ipfs/js-ipfs?label=help%20wanted
```

Comment:
Since Waffle is shutting down, should we start linking to GH issues instead?

E.g. the super-memorable https://github.com/ipfs/js-ipfs/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22

Reply:
Yes, this needs a PR to https://github.com/ipfs/js-ipfs/blob/master/RELEASE.md also